doc/source/releases/24.0_announce.rst

```diff
@@ -2,7 +2,7 @@
 :orphan:
 
 ===========================================================
-24.0 Galaxy Release
+24.0 Galaxy Release (Early 2024)
```

Comment:
I think "24.0 Galaxy Release" would be sufficient. "Early" is vague (and I understand, it's by design). But, to reiterate, given that it doesn't adjust well when changing release frequency, as well as our (occasional) tendency to push back the release date weeks if not months, I'd keep it simple - 24.0. Besides, the "early" term feels almost like "early release", as in preliminary.

Reply:
I like this as-is -- stating to expect the release "Early 2024", not that this nomenclature is a part of the release name.